app/views/filters/index.html.erb

```diff
@@ -41,6 +45,8 @@
           <%= action_buttons(
                     display_link_if_authorized(_("Edit"), hash_for_edit_filter_path(:id => filter, :role_id => @role).
                             merge(:auth_object => filter, :authorizer => authorizer)),
+                    display_link_if_authorized(_("Disable overriding"), hash_for_disable_overriding_filter_path(:id => filter, :role_id => @role).
```

Comment:
This action should not show up if `filter.override` is false, it doesn't do anything